### PR TITLE
[Docs] Add `WKT` function

### LINKED ISSUE #2040: Document `wkt` function.
Create documentation for the `wkt` function. See #1833 for details on how to complete this issue.

Child issue of #1833.

### PR DESCRIPTION
**Pull request by Blargian**
Add WKT function to documentation as part of the project to document functions missing in the documentation. Closes [2040](https://github.com/ClickHouse/clickhouse-docs/issues/2040).

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Documentation entry for user-facing changes

- [X] Documentation is written (mandatory for new features)

### CODE REVIEW

File: docs/en/sql-reference/functions/geo/polygon.md [hunk 1]
```diff
@@ -4,6 +4,34 @@ sidebar_label: Polygons
 title: "Functions for Working with Polygons"
 ---
 
+## WKT
+
+Returns a WKT (Well Known Text) geometric object from tuples which represent point coordinates.
```
Comment (alexey-milovidov) [inline docs/en/sql-reference/functions/geo/polygon.md:9]: This documentation is incomplete - we need to explain behavior for various geometric types and various shapes of arrays.

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit e1b2e0a00bae0347f02eba4c484577ed41b51c46 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/61478/e1b2e0a00bae0347f02eba4c484577ed41b51c46/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/sql-reference/functions/geo/polygon.md (hunk 1 above)
Comment (Blargian) [reply docs/en/sql-reference/functions/geo/polygon.md]: @alexey-milovidov thanks, I have pushed the update. How does it look now?

File: docs/en/sql-reference/functions/geo/polygon.md [hunk 2]
```diff
@@ -4,6 +4,62 @@ sidebar_label: Polygons
...
+
+**Syntax**
+
+```sql
+WKT(points)
+```
+
+**Parameters**
+
+- `points`: a [tuple](../../data-types/tuple.md) representing coordinate point (x, y) or an [array](../../data-types/array.md) of tuples, or an array of multi-dimensional tuple arrays.
```
Comment (alexey-milovidov) [inline docs/en/sql-reference/functions/geo/polygon.md:23]: There are also Geo data types: https://clickhouse.com/docs/en/sql-reference/data-types/geo

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**